release-announcements/20210701-yt-4.0.0.txt

```diff
@@ -0,0 +1,81 @@
+
+Information about "what's new" can be found at: https://yt-project.org/docs/4.0.0/yt4differences.html .
+
+A set of highlights of the pull requests that went into this release can be found here:
+
+https://gist.github.com/matthewturk/7a1f21d98aa5188de7645eda082ce4e6#file-highlights-md
+
+A full listing of the pull requests that went into this release can be found here:
+
+https://gist.github.com/matthewturk/7a1f21d98aa5188de7645eda082ce4e6#file-yt4-changelog-md
+
+This release has been years in the making, and features contributions from dozens of individuals, many of whom were new to the project.  It marks the culmination of a set of wide-reaching infrastructure changes that result in much higher fidelity to the underlying data for particle datasets, and we anticipate that improvements to the overall project and project infruastructure will lead directly to a much more rapid release cycle of enhancements and bug fixes in the future.
+
+At the end of this email you can find a non-exhaustive list of individuals who have contributed to this release; many of them made their first contribution during this release!
+
+Binaries for yt 4.0 are available via pip and conda. If you use conda to manage your Python installation, you can update yt via:
+
+$ conda update -c conda-forge yt
+
+Conda-forge package updates may not be up quite yet but should be uploaded by the end of the day.
+
+If you manage your Python installation with pip, you can update yt via:
+
+$ python -m pip install -U yt
+
+For more information, including installation instructions, links to community resources, and information on contributing to yt’s development, please see the yt homepage at https://yt-project.org and the documentation for yt-4.0 at http://yt-project.org/docs/4.0.0
+
+We especially thank Nathan Goldbaum for his many years of work and contributions to this release and to the yt community.
+
+yt is the product of a large community of developers and users and we are extraordinarily grateful for and proud of their contributions. Please forward this announcement on to any interested parties.
+
+Thank you,
+
+The yt team
+
+Contributors to this release:
+
+ * Ashley Kelly
+ * Axel Huebl
+ * Baptiste Mouginot
+ * Ben Kimock
+ * Bili Dong
+ * Brandon Qiao
+ * Britton Smith
+ * Cameron Hummels
+ * Camille Avestruz
+ * Chris Havlin
+ * Christopher W. Evans
+ * Clayton Strawn
+ * Clément Robert
+ * Corentin Cadiou
+ * David Grote
+ * Desika Narayanan
+ * Jared Coughlin
+ * Jody Klymak
+ * John Regan
+ * John ZuHone
+ * Kacper Kowalik
+ * Kelton Halbert
+ * Madicken Munk
+ * Matthew Abruzzo
+ * Matthew Turk
+ * Matthias Bussonnier
+ * Meagan Lang 
+ * Michael Zingale
+ * Miguel de Val-Borro
+ * Nathan Goldbaum
+ * Neïl Zaim
+ * Niels Claes
+ * Ole Streicher
+ * Philipp Edelmann
+ * Revathi Jambunathan
+ * Shea Garrison-Kimmel
+ * Stefan Arridge
```

Comment:
I ran a script to detect new comers and highlight them (as was done in at least some previous release announcements)
```suggestion
Ashley Kelly
Axel Huebl *
Baptiste Mouginot *
Ben Kimock *
Bili Dong
Brandon Qiao
Britton Smith
Cameron Hummels
Camille Avestruz *
Chris Havlin *
Christopher W. Evans *
Clayton Strawn
Clément Robert
Corentin Cadiou
David Grote *
Desika Narayanan *
Jared Coughlin *
Jody Klymak *
John Regan
John ZuHone
Kacper Kowalik
Kelton Halbert *
Madicken Munk
Matthew Abruzzo *
Matthew Turk
Matthias Bussonnier *
Meagan Lang *
Michael Zingale
Miguel de Val-Borro
Nathan Goldbaum
Neïl Zaim *
Niels Claes
Ole Streicher *
Philipp Edelmann *
Revathi Jambunathan *
Shea Garrison-Kimmel *
Stefan Arridge *
```

note that the result of my script is not perfect and may incorrectly highlight people who are actually returning contributors. For instance, Niels Claes was not mentioned in previous announcements, most likely because we skipped the 3.6.0 release where he wrote his _actual_ first contributions.

Reply:
for reference here's my script. `list.txt`'s content was copy pasted from the current state of this PR
```python
from glob import glob

contributors_4 = [_.strip().removeprefix("* ") for _ in open("list.txt").readlines()]
corpus = [open(file).read() for file in glob("release-announcements/*txt") if file]
assert corpus
full_history = "".join(corpus)
for contributor in contributors_4:
    if contributor not in full_history:
        print(f"{contributor} *")
    else:
        print(contributor)
```